content/en/profiler/enabling/ddprof.md

```diff
@@ -25,7 +25,7 @@ The Datadog Continuous Profiler for compiled languages (`ddprof`) uses OS level
 ## Requirements
 
 Supported operating systems
-: Linux kernel v4.17+, libc 2.18+
+: Linux kernel v4.17+, libc 2.18+ or musl libc
```

Comment:
@nsavoire raised a good point in one of the comments, which I didn't really think about before. 
 *libc* refers to the ISO C standard library, which is "versioned" (IIRC, only additively) alongside the C language standard (e.g., C11, etc).  *glibc* refers to the GNU implementation of the ISO C standard library, and it is versioned in the way we refer to here.

Since we're making a statement about glibc (or at least, glibc _compatibility_), should we be specific and write `glibc` rather than `libc`?

Reply:
true